datafusion/expr/src/logical_plan/plan.rs

```diff
@@ -1390,8 +1399,9 @@ pub enum Partitioning {
     RoundRobinBatch(usize),
     /// Allocate rows based on a hash of one of more expressions and the specified number
     /// of partitions.
-    /// This partitioning scheme is not yet fully supported. See <https://issues.apache.org/jira/browse/ARROW-11011>
```

Comment:
This issue (https://github.com/apache/arrow-datafusion/issues/131) has been closed so I removed the comment